packages/tabs/src/LionTabs.js

```diff
@@ -161,26 +160,36 @@ export class LionTabs extends LitElement {
     this.selectedIndex = 0;
   }
 
+  /** @param {import('lit-element').PropertyValues } changedProps */
   firstUpdated(changedProps) {
     super.firstUpdated(changedProps);
     this.__setupSlots();
   }
 
   __setupSlots() {
-    const tabSlot = this.shadowRoot.querySelector('slot[name=tab]');
-    const handleSlotChange = () => {
-      this.__cleanStore();
-      this.__setupStore();
-      this.__updateSelected(false);
-    };
-    tabSlot.addEventListener('slotchange', handleSlotChange);
+    if (this.shadowRoot) {
+      const tabSlot = this.shadowRoot.querySelector('slot[name=tab]');
+      const handleSlotChange = () => {
+        this.__cleanStore();
+        this.__setupStore();
+        this.__updateSelected(false);
+      };
+
+      if (tabSlot) {
+        tabSlot.addEventListener('slotchange', handleSlotChange);
+      }
+    }
   }
 
   __setupStore() {
     /** @type {StoreEntry[]} */
     this.__store = [];
-    const buttons = this.querySelectorAll('[slot="tab"]');
-    const panels = this.querySelectorAll('[slot="panel"]');
+    const buttons = /** @type {HTMLElement[]} */ (Array.from(
```

Comment:
Type casting to HTMLElement[] because the default is Element[], but then we get problems further down the line